forwarder/writer.go

```diff
@@ -63,19 +75,34 @@ func NewForwarder(config *Config) *Forwarder {
 				Buckets:   prometheus.DefBuckets,
 			},
 		),
+		reqTimeseriesCount: prometheus.NewHistogram(
+			prometheus.HistogramOpts{
+				Namespace: namespace,
+				Subsystem: subsystem,
+				Name:      "request_timeseries_count",
```

Comment:
Histogram will automatically create the metrics `request_timeseries_count_count` and `request_timeseries_count_sum` which looks strange.

Maybe `request_timeseries_length` instead.